core/src/config/networks/ethereum/assets.ethereum.mainnet.json

```diff
@@ -489,7 +547,10 @@
       "imageUrl": "https://assets.coingecko.com/coins/images/12681/small/UST.png?1601612407",
       "name": "Terra USD",
       "network": "ethereum",
-      "symbol": "UST"
+      "symbol": "UST",
+      "homeNetwork": "ethereum",
+      "decommissioned": true,
+      "decommissionReason": "Temporarily decommissioned due to Terra instability"
```

Comment:
Is it still decomissioned?

Reply:
good catch @pgoos, I had restored a previous config that had it decomissioned. thanks :tada: